ioctl/newcmd/action/action.go

```diff
@@ -217,10 +217,12 @@ func NewActionCmd(client ioctl.Client) *cobra.Command {
 	// cmd.AddCommand(NewActionDeposit(client))
 	// cmd.AddCommand(NewActionSendRaw(client))
 
-	client.SetEndpointWithFlag(ac.PersistentFlags().StringVar)
-	client.SetInsecureWithFlag(ac.PersistentFlags().BoolVar)
+	RegisterWriteCommand(client, cmd)
```

Comment:
`RegisterWriteCommand` is just used in some sub commands, not in all sub commands.